public/humans.txt

```diff
@@ -5,76 +5,88 @@ Oak National Academy’s tech platform has been created by a team of human devel
 /* TEAM */
 
 
+  Amy Matthews (Infrastructure Engineer)
+  From: Kent, UK
+
+  Andrew Woods (Software Engineering Manager)
+  Site: https://andrewwoods.super.site/
+  LinkedIn: https://www.linkedin.com/in/andykwoods/
+  From: Dublin, Ireland
+
+  Ben Protheroe (Software Engineer)
+  From London, lives in Cornwall
+
+  Dan Dupont (Product Manager)
+  LinkedIn: https://www.linkedin.com/in/dandupontlopez/
+  From: Bristol, UK
+
+  Dorothy Yau (Software Engineer)
+  LinkedIn: https://www.linkedin.com/in/dorothyyau/
+  From: London, UK
+
+  Helen Harlow (Product Designer)
+  LinkedIn: https://www.linkedin.com/in/helen-harlow/
+  From: Norwich, UK
+
   Ian Kynnersley (Head of Engineering)
   Site: https://www.iankynnersley.co.uk/
   LinkedIn: https://www.linkedin.com/in/iankynnersley/
   Twitter: https://twitter.com/kpopper
   From: London, UK
   
-
-  Jack Skerrett (QA)
+  Jack Skerrett (Quality Assurance Engineer)
```

Comment:
My official title is "Quality Assurance Analyst"

Reply:
updated